tests/interpreters/test_dart.py

```diff
@@ -0,0 +1,293 @@
+from m2cgen import ast
```

Comment:
Please add some more tests related to the `BinExpressionDepthTrackingMixin`. You can use the following code as example:
https://github.com/BayesWitnesses/m2cgen/blob/f14114614113d018531b7232750cc6ed977272e8/tests/interpreters/test_r.py#L220-L361
https://github.com/BayesWitnesses/m2cgen/blob/f14114614113d018531b7232750cc6ed977272e8/tests/interpreters/test_python.py#L187-L312
https://github.com/BayesWitnesses/m2cgen/blob/f14114614113d018531b7232750cc6ed977272e8/tests/interpreters/test_java.py#L355-L521